pkg/checkpoint/pod.go

```diff
@@ -66,14 +66,17 @@ func sanitizeCheckpointPod(cp *v1.Pod) *v1.Pod {
 	// Remove affinity
 	cp.Spec.Affinity = nil
 
+	// Remove node selector
+	cp.Spec.NodeSelector = nil
+
```

Comment:
This is the fix.

Reply:
whoops i didnt realize the number of changes that would be required.